gui_application.py

```diff
@@ -0,0 +1,256 @@
+# Project Name:  SNAKE PEOPLE UML Editor
+# File Name:     gui_application.py
+
+# Imports:
+import json
+import jsonpickle
```

Comment:
The json pickle import can be removed right?

Reply:
yea i just had them there cause i copied the imports from the other files. they can be removed